helm/alfresco-process-application/values.yaml

```diff
@@ -85,6 +85,10 @@ activiti-cloud-query:
       value: "false"
     - name: "ACTIVITI_CLOUD_APPLICATION_NAME"
       value: "{{ .Release.Name }}"
+    - name: "GRAPHIQL_GRAPHQL_WS_PATH"
```

Comment:
can you check if the deployment-service adds them? if not we need a ticket for that too

Reply:
I already checked, and I'm doing it in another PR